openid-4-verifiable-presentations-1_0.md

```diff
@@ -432,99 +415,23 @@ wallet_metadata=%7B%22vp_formats_supported%22%3A%7B%22jwt_vc_json%22%3A%7B%22alg
-
-The formats supported by a Verifier may be set up using the metadata parameter `vp_formats` (see (#client_metadata_parameters)). The Wallet MUST ignore any `format` property inside a `presentation_definition` object if that `format` was not included in the `vp_formats` property of the metadata.
-
-Note: When a Verifier is requesting the presentation of a Verifiable Presentation containing a Verifiable Credential, the Verifier MUST indicate in the `vp_formats` parameter the supported formats of both Verifiable Credential and Verifiable Presentation.
-
-## `presentation_definition_uri` Parameter {#request_presentation_definition_uri}
-
-`presentation_definition_uri` is used to retrieve the Presentation Definition from the resource at the specified URL, rather than being passed by value. The Wallet MUST send an HTTP GET request without additional parameters. The resource MUST be exposed without further need to authenticate or authorize. 
-
-The protocol for the `presentation_definition_uri` MUST be HTTPS.
-
-The following is a non-normative example of an HTTP GET request sent after the Wallet received `presentation_definition_uri` parameter with the value `https://server.example.com/presentationdefs?ref=idcard_presentation_request`:
-
-```
-GET /presentationdefs?ref=idcard_presentation_request HTTP/1.1
-Host: server.example.com
-```
-
-The following is a non-normative example of an HTTP GET response sent by the Verifier in response to the above HTTP GET request:
-
-```
-HTTP/1.1 200 OK
-...
-Content-Type: application/json
-
-{
-  "id": "vp token example",
-  "input_descriptors": [
-    {
-      "id": "id card credential",
-      "format": {
-        "ldp_vc": {
-          "proof_type": [
-            "Ed25519Signature2018"
-          ]
-        }
-      },
-      "constraints": {
-        "fields": [
-          {
-            "path": [
-              "$.type"
-            ],
-            "filter": {
-              "type": "string",
-              "pattern": "IDCardCredential"
-            }
-          }
-        ]
-      }
-    }
-  ]
-}
-```
 
 ## Using `scope` Parameter to Request Verifiable Credential(s) {#request_scope}
 
 Wallets MAY support requesting presentation of Verifiable Credentials using OAuth 2.0 scope values.
 
-Such a scope value MUST be an alias for 
-- a well-defined DCQL query, or
-- a well-defined Presentation Definition (for [@!DIF.PresentationExchange]) that will be referred to in the `presentation_submission` response parameter. 
+Such a `scope` parameter value MUST be an alias for a well-defined DCQL query. Since multiple `scope` values can be used at the same time, the identifiers for credentials (see (#credential_query)) and claims (see (#claims_query)) within the DCQL queries associated with `scope` values MUST be unique.
```

Comment:
I wrote a whole comment on not understanding this requirement, but I think I understand now. The response contains only the DCQL query id as keys, so if you reuse the same ID across scope values it means you don't know which scope was used for the DCQL presentation. 

I think it could benefit from a small clarification, as it took me quite a while to understand the reasoning for this.

Reply:
I'll add a clarification!